lib/question.js

```diff
@@ -1894,6 +1896,59 @@ module.exports = {
             callback(null);
           });
         },
+        (callback) => {
+          // if barcodes found, get available pdf files
+          const barcodedSubmissions = locals.submissions
+            .slice(Math.max(locals.submissions.length - 5, 0)) // only last 5 displayed
```

Comment:
In my latest test, more than 5 submissions can be displayed on a question element. I am not too sure if we need this math max bit of code any longer. Has the max 5 displayed submissions rule changed?

Reply:
All submissions are rendered into the page; some (currently all but the 3 most recent) are just hidden inside a collapsible panel. We'll need to fetch PDFs for all submissions here, not just the first 5.